iceoryx_posh/include/iceoryx_posh/internal/popo/ports/subscriber_port_user.hpp

```diff
@@ -91,7 +91,7 @@ class SubscriberPortUser
 
     MemberType_t* m_subscriberPortDataPtr;
 
-    ChunkReceiver m_chunkReceiver;
+    ChunkReceiver<SubscriberPortData::ChunkQueueData_t> m_chunkReceiver;
```

Comment:
symmetric to the sender side would be then
    `using ChunkQueue_t = ChunkQueue<SubscriberPortData::ChunkQueueData_t>;`
    `ChunkReceiver<ChunkQueue_t > m_chunkReceiver;`

Reply:
There's no `ChunkQueue` right now. There's only the queue popper. Should we rework the structure in EA?

Reply:
Also here then using `ChunkQueuePopper_t = ChunkQueuePopper<SubscriberPortData::ChunkQueueData_t>;` 

Sorry for confusing

Reply:
Done